[DOCS] DSB-484 Improve Quickstart diagram links (#8951)
great-expectations/great_expectations: docs/docusaurus/docs/tutorials/quickstart/quickstart.md

## Data validation workflow

> The following diagram illustrates the end-to-end GX data validation workflow implemented by the quickstart code. Click a workflow step to view related code.

Reviewer: ```suggestion
The following diagram illustrates the end-to-end GX data validation workflow that you'll implement with this quickstart. Click a workflow step to view the related content.
```

## Related documentation

> If you're ready to continue your GX journey, the following topics can help you implement a tailored solution for your specific environment and business requirements:

Reviewer: ```suggestion
If you're ready to continue your GX journey, the following topics can help you implement a solution for your specific environment and business requirements:
```
